### PR TITLE
Update Solana Provider Package Readme

### PR DESCRIPTION
**Pull request by shahbaz17**

### CODE REVIEW

**Review by yashovardhan — COMMENTED** [review]
@shahbaz17 did you try running this locally once? Does this return the provider as expected?

File: packages/providers/solana-provider/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Web3Auth Solana Provider can be used to interact with wallet or connected Solana
 
 ## 📖 Documentation
 
-Read more about Web3Auth Ethereum Provider in the [official Web3Auth Documentation](https://web3auth.io/docs/sdk/web/providers/solana).
+Read more about Web3Auth Ethereum Provider in the [official Web3Auth Documentation](https://web3auth.io/docs/sdk/web/providers/solana#getting-a-provider-from-any-secp512k1-private-key).
```
Comment (yashovardhan) [inline packages/providers/solana-provider/README.md:12]: This says ethereum provider... we need to change it to Solana Provider

Also Solana Provider gives more than just private key to provider right? We also have the general flow where Solana Wallet is used.. we should document both the cases

Comment (shahbaz17) [reply packages/providers/solana-provider/README.md]: I have updated it to Solana.

https://web3auth.io/docs/sdk/web/providers/solana uses Solana Wallet now to be consistent with the Connect Blockchain page. BTW what's the other flow?
@yashovardhan

**Comment by shahbaz17** [comment]
@yashovardhan 

Updated the example code with a working code.

**Comment by himanshuchawla009** [comment]
plz squash them in a single commit.

**Review by yashovardhan — APPROVED** [review]
LGTM

**Review by yashovardhan — APPROVED** [review]
LGTM

**Comment by shahbaz17** [comment]
> plz squash them in a single commit.

@himanshuchawla009 made it into a single commit.

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**